pkgs/development/python-modules/fsspec/default.nix

```diff
@@ -33,6 +28,9 @@ buildPythonPackage rec {
     # resolved in https://github.com/intake/filesystem_spec/issues/432 and
     # can be enabled again from version 0.8.4
     "test_pathobject"
```

Comment:
Can this be removed?

Reply:
I removed it. The issue has indeed been resolved on the fsspec side.